upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -542,6 +542,69 @@ There you can find links to upgrade notes for other versions too.
         ```
         - for more information you can see the [PR](https://github.com/shopsys/shopsys/pull/1461)
 
+
+- add graphic checkboxes and radiobuttons ([#1465](https://github.com/shopsys/shopsys/pull/1465))
+    - update following files
+    - `project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/box/chooser.less` (line 18)
+        ```diff
+          align-items: flex-start;
+        + cursor: pointer;
+
+          &__check {
+        ```
+    - `project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/form/choice.less` (line 4)
+        ```diff
+          display: inline-block;
+        - padding-left: 20px;
+          line-height: 20px;
+        ```
+
+    - `project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/in/icon.less` (line 13)
+        ```diff
+          font-size: 16px;
+        + height: 16px;
+          cursor: help;
+        ```
+
+    - add new file `project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/core/form/custom-inputs.less` according to pull request
```

Comment:
Could you add link to file here please? `https://github.com/shopsys/shopsys/blob/8.1/project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/core/form/custom-inputs.less`

Reply:
added